docs/extensibility.md

```diff
@@ -1,364 +1,148 @@
 # Extensibility
 
-Akri has [implemented several discovery protocols](./roadmap.md#currently-supported-protocols) with sample brokers and applications. However, there may be protocols you would like to use to discover resources that have not been implemented yet.  This document walks you through how to **extend Akri** to discover new types of devices that you are interested in.
+Akri has [implemented several discovery protocols](./roadmap.md#currently-supported-protocols) with sample brokers and applications. However, there may be protocols you would like to use to discover resources that have not been implemented yet.  To enable the discovery of resources via a new protocol, you much implement a Discovery Handler (DH), which does discovery on behalf of the Agent. A Discovery Handler is anything that implements the `Discovery` service and `Registration` client defined in the [Akri's discovery gRPC proto file](../discovery-utils/proto/discovery.proto). These DHs run as their own Pods and are expected to register with the Agent, which hosts the `Registration` service defined in the gRPC interface. A discovery handler can be written in any language using protobuf; however, Akri has provided a template for accelerating creating a discovery handler in Rust.  
```

Comment:
`you much implement` ===> `you must implement`

Reply:
fixed in 1b8bd58